source/common/http/filter/ratelimit.cc

```diff
@@ -117,9 +117,16 @@ void Filter::complete(Envoy::RateLimit::LimitStatus status) {
     break;
   case Envoy::RateLimit::LimitStatus::OverLimit:
     cluster_->statsScope().counter("ratelimit.over_limit").inc();
-    Http::CodeUtility::ResponseStatInfo info{config_->scope(), cluster_->statsScope(), EMPTY_STRING,
-                                             *getTooManyRequestsHeader(), true, EMPTY_STRING,
-                                             EMPTY_STRING, EMPTY_STRING, EMPTY_STRING, false};
+    Http::CodeUtility::ResponseStatInfo info{config_->scope(),
```

Comment:
Best to disable the header ordering in `clang-format`. I recall looking into whether we could get it to do what we wanted and concluded no. See https://github.com/lyft/envoy/pull/793. I think it had to do with the block splitting etc. Might be better in 5.0, not sure.